source/Planet.cpp

```diff
@@ -639,7 +640,10 @@ void Planet::DeployDefense(list<shared_ptr<Ship>> &ships) const
 		return;
 
 	auto end = defenders.begin();
-	defenseFleets[defenseDeployed]->Enter(*GetSystem(), defenders, this);
+	if(defenseFleets[defenseDeployed]->IsValid())
+		defenseFleets[defenseDeployed]->Enter(*GetSystem(), defenders, this);
+	else
+		Logger::LogError("Warning: skipped an undefined tribute fleet of planet \"" + name + "\".");
```

Comment:
can we include that the fleet could also be only defined in parts/missing only one thing?

Reply:
"not fully defined" is the language used in when issuing the warnings on loading, and I think "incomplete" is in line with that.